GSA/digitalgov.gov
# Updates to Multilingual topic page and Bilingual Glossaries page

## Summary

- Updated Bilingual Glossaries, Dictionaries, and Style Guides page to include link to new health terms glossary
- Fixed link formatting on one resource
- Updated featured resource, legislation card, and essential knowledge on Multilingual topic page
- Updated link to 2022 DOJ memo and removed button formatting

## Preview

[Multilingual topic page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/mh-multilingual-glossary-updates/topics/multilingual/)
[Bilingual Glossaries, Dictionaries, and Style Guides](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/mh-multilingual-glossary-updates/resources/bilingual-glossaries-dictionaries-style-guides/)
[Updated link to 2022 DOJ memo](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/mh-multilingual-glossary-updates/resources/dept-of-justice-memo-reaffirming-the-mandates-of-eo-13166/)




## Solution

Provide a summary of the solution this PR offers.




## How To Test

1. Check all link formatting
2. Make sure links work
